python_scripts/03_categorical_pipeline.py

```diff
@@ -52,38 +53,37 @@
 data["native-country"].value_counts().sort_index()
 
 # %% [markdown]
-# One might question how could we easily recognize categorical columns. One
-# useful tip is to look at the data type of each column.
+# Now the question is: how can we easily recognize categorical columns
+# among the dataset ? Part of the answer lies in the columns' data type:
 
 # %%
 data.dtypes
 
 # %% [markdown]
-# If we look at the "native-country" column data type, we observe it is of
-# `object` data type. The reason is that categories are represented with
```

Comment:
Hmmm maybe keep this, some people will not be very familiar with numpy. Adding string may be make it more clear than only dtype object